deployment/postgres-init-db/sql/ui/init.sql

```diff
@@ -1,4 +1,8 @@
 begin;
```

Comment:
Where do the UI files come from? I know for the other databases, they're filled in by the `DistributeSQL` Gradle task, but something feels weird with the UI DB only being defined in `deployment`.

Reply:
The UI files are, in spirit, owned by the `aerie-ui` repo - so there is no corresponding gradle project for them in this repo.

The question of which things belong in `deployment`, vs in other subprojects is a good one - we used to keep everything in deployment, and then we decided to try to communicate ownership of the sql files by moving them into their corresponding project files. Our intentions were good, but `distributeSql` has been the source of much confusion, so perhaps it would be best to go back to keeping everything in deployment 🤷 

For the purposes of this PR, we're leaving `distributeSql` alone, and putting migrations in the locations expected by hasura. Fortunately, this is something that we do have the flexibility to change after the fact :+1:

Reply:
Oh yeah, that is a weird situation, although I think it's mostly weird because Aerie's a mono repo _except_ for the Gateway/UI. Since it doesn't sound like anything in the UI repo is affecting/sharing ownership over these files, though, I'm no longer concerned.